fixtures/app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "test": "eslint --ext js,ts,jsx,tsx,mjs ."
+  },
+  "devDependencies": {
+    "@vtex/prettier-config": "^1.0.0",
+    "eslint": "^7",
+    "eslint-config-vtex": "^14.1.2",
```

Comment:
Shouldn't be `15.0.0`? Same to `eslint-config-vtex-react`.

Reply:
there is a step in CI to only install the packages that were changes. I did it like this so the developer could test locally using the published versions